doc/userDocs/usersGuide.md

```diff
@@ -141,7 +140,24 @@ The image below shows the lines that need to be edited in the configuration file
 2. Update *kubeconfigFile* to the full path to your kubeconfig file.  The paths will be identical if using a single cluster for both the app and driver.
 3. Update *kubeconfigContext* parameters with the name of the context for your cluster from the kubeconfig file, such as `kubernetes-admin@kubernetes`.
     - Set an empty string (`""`) to use the current-context defined in the kubeconfigFile.
-4. Set *useLoadBalancer* to `true` if your cluster supports LoadBalancer services. Otherwise, set it to `false` to use a NodePort for ingress.
+4. Choose a value for *appIngressMethod* that reflects the method you want to use
+  for communication between the workload drivers and the applications. You can choose 
+  from among:
+    - `clusterip`: Traffic from the workload drivers to the applications will use a 
```

Comment:
I notice later in the doc you adopt the ordering "loadbalancer", "nodeport" and "clusterip". I like having the default listed first and cluster IP last as it requires only one cluster and doesn't stress the entire network stack, so could you put clusterip last.

Reply:
That ordering makes sense.  However the reason that I put clusterip first here is that it made it easier to explain that you might want to use loadbalancer or nodeport even on a single cluster.  

I will try putting clusterip last and rewording the others slightly.  Let me know what you think.